src/app/core/session/session-service/remote-session.spec.ts

```diff
@@ -48,5 +62,33 @@ describe("RemoteSessionService", () => {
     expect(service.loginState.value).toBe(LoginState.UNAVAILABLE);
   });
 
+  it("should retry auto-login if fetch fails and fetch again", async () => {
+    const initSpy = spyOn(service["database"], "initIndexedDB");
+    spyOn(PouchDB, "fetch").and.returnValues(
+      Promise.resolve({
+        status: HttpStatusCode.Unauthorized,
+        ok: false,
+      } as Response),
+      Promise.resolve({ status: HttpStatusCode.Ok, ok: true } as Response)
+    );
+    let calls = 0;
+    mockAuthService.addAuthHeader.and.callFake((headers) => {
+      headers.Authorization = calls++ === 1 ? "active" : "inactive";
```

Comment:
I trust you, but I have no clue what this means - what's "active" of "inactive" as a header value for "Authorization"?